Library/Formula/boot.rb

```diff
@@ -0,0 +1,18 @@
+require "formula"
+
+class Boot < Formula
+  homepage "http://boot-clj.com"
+  version "2.0.0-rc4"
```

Comment:
Similar to Netflix's denominator project this `boot.sh` file just wraps a jar that loads other dependencies from Maven. The script itself is very stable itself. Is that ok? 

If not, are there other ways to get this into Homebrew before it's 2.0.0 final?